packages/storybook/src/spright/rectangle/rectangle-matrix.stories.ts

```diff
@@ -0,0 +1,59 @@
+import type { StoryFn, Meta } from '@storybook/html';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { rectangleTag } from '@ni/spright-components/dist/esm/rectangle';
+import {
+    createMatrix,
+    sharedMatrixParameters,
+    createMatrixThemeStory,
+    cartesianProduct,
+    createMatrixInteractionsFromStates
+} from '../../utilities/matrix';
+import { createStory } from '../../utilities/storybook';
+import { hiddenWrapper } from '../../utilities/hidden';
+import {
+    disabledStates,
+    type DisabledState,
+    disabledStateIsEnabled
+} from '../../utilities/states';
+
+const metadata: Meta = {
+    title: 'Tests (Spright)/Rectangle',
```

Comment:
We have logic in `manager.js` to exclude the tests folder from public storybook builds and I think it won't match the "(Spright)" pattern.

How does it look if we do "Tests/Spright/Rectangle"? Or "Tests/Spright Rectangle"? Or we can update the logic to match this pattern too.

Reply:
I'd vote for `Tests Spright/Rectangle` to avoid extra punctuation since it's used for links etc

Reply:
I did the simplest thing, which is to change `startsWith('Tests/')` to `startsWith('Tests')`. I'm not worried about potential, future false positives like `Testsomething`, but if anyone is, I can change it to use a regex.

Reply:
I'd still like to avoid punctuation in the name

Reply:
Sure, I hadn't refreshed to see your reply before I responded to Jesse.